tket/src/Circuit/CircPool.cpp

```diff
@@ -910,6 +914,23 @@ Circuit TK2_using_CX(const Expr &alpha, const Expr &beta, const Expr &gamma) {
   }
 }
 
+Circuit TK2_using_CX_optimal(
```

Comment:
I would suggest renaming this function to `TK2_using_CX` and the other `TK2_using_CX` to `normalised_TK2_using_CX`. This way we can keep consistency between the names here and in `pytket.circuit.library`.

Reply:
Good idea.

Reply:
Done.